test/tests/api/v2_0/skupack_tests.py

```diff
@@ -0,0 +1,420 @@
+from config.api2_0_config import *
+from modules.logger import Log
+from on_http_api2_0 import ApiApi as Api
+from on_http_api2_0.rest import ApiException
+from proboscis.asserts import *
+from proboscis import SkipTest
+from proboscis import test
+from json import dumps, loads
+from on_http_api2_0 import rest
+import time
+import requests
+import os
+import subprocess
+import json
+import tarfile
+import shutil
+
+
+LOG = Log(__name__)
+
+
+@test(groups=['skus_api2.tests'])
+class SkusTests(object):
+    def __init__(self):
+
+        self.__client = config.api_client
+        self.clear()
+        self.__nodes= []
+        self.__sku_id = ""
+        self.__workflows = {}
+        self.__tasks ={}
+        self.__config_json = {}
+        self.__rootDir ="/tmp/tarball/"
+        self.__skuPackTarball =  self.__rootDir +"mytest.tar.gz"
+
+    def __get_data(self):
+        return loads(self.__client.last_response.data)
+
+    @test(groups=['api2_get-skus'])
+    def test_skus(self):
+        """Testing GET:api/2.0/skus to get list of skus"""
+        Api().skus_get()
+        rsp = self.__client.last_response
+        assert_not_equal(404, rsp.status, message=rsp.reason)
+        assert_equal(200, rsp.status, message=rsp.reason)
+        data = loads(self.__client.last_response.data)
```

Comment:
data isn't used?  Might be worth asserting on something in the data.  Additionally, __get_data() can be used rather than duplicating the code.